sessions/src/test/kotlin/io/spine/examples/pingh/sessions/SessionsContextSpec.kt

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2024, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.examples.pingh.sessions
+
+import io.spine.base.EventMessage
+import io.spine.examples.pingh.sessions.event.UserLoggedIn
+import io.spine.examples.pingh.sessions.event.UserLoggedOut
+import io.spine.examples.pingh.sessions.given.userSession
+import io.spine.examples.pingh.sessions.given.logUserOut
+import io.spine.examples.pingh.sessions.given.logUserIn
+import io.spine.examples.pingh.sessions.given.sessionId
+import io.spine.examples.pingh.sessions.given.sessionIdBy
+import io.spine.server.BoundedContextBuilder
+import io.spine.testing.server.EventSubject
+import io.spine.testing.server.blackbox.ContextAwareTest
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+
+@DisplayName("Sessions Context should")
+public class SessionsContextSpec : ContextAwareTest() {
+
+    protected override fun contextBuilder(): BoundedContextBuilder =
+        newBuilder()
+
+    @Nested
+    public inner class `handle 'LogUserIn' command` {
```

Comment:
To make the display names reading as a sentence (when concatenating all levels), it such intermediate display names should have something like "... and". 

E.g. `Sessions Context should` + `handle 'LogUserIn' command, and` + `emit 'UserLoggedIn' event` would make a sentence. Without "and" it just does not read.

Please address throughout the tests.